mod_root/BepInEx/plugins/EpicLoot/adventuredata.json

```diff
@@ -189,8 +172,8 @@
     "GoldMinLevel": 3,
     "GoldMaxLevel": 3,
     "GoldHealthMultiplier": 3.0, 
-    "AddsMinLevel": 1,
-    "AddsMaxLevel": 2,
+    "AddsMinLevel": 2,
+    "AddsMaxLevel": 4,
```

Comment:
What is this change for?

Reply:
was messing around trying to get it do detect new content. Anyway this is the level of the adds for the mobs that come with the bounty target. I am going to repush with the previously modified values of `2` and `5` respectively.